SPECS/aws-nitro-enclaves-cli.spec

```diff
@@ -136,12 +136,34 @@ echo -e "
 
 
 %postun
-# Remove any directory which was created by the driver as well as unload the driver
-rm -f /usr/lib/modules-load.d/nitro_enclaves.conf
-rm -f /usr/lib/udev/rules.d/99-nitro_enclaves.rules
-rm -f /usr/lib/tmpfiles.d/nitro_enclaves.conf
-rm -rf %{ne_run_dir}
-rm -rf %{ne_log_dir}
+if [ $1 -ne 1 ]; then
```

Comment:
I think also on install you wouldn't want to remove all those file which people might have installed by hand.

Reply:
I don't think the `%postun` scriptlet is run on package installation.